internal/ui/logger.go

```diff
@@ -14,7 +14,13 @@ func init() {
 		TimestampFormat: "2006-01-02 15:04:05", // Custom format
 		FullTimestamp:   true,
 	})
-	level, err := log.ParseLevel(util.GetEnvOrDefault(constant.LogLevelKey, "info"))
+	var logLevel string
```

Comment:
Why this change?

Reply:
There was a circular dependency. Utils was using a logger, and the logger was using a util.
It was found that there were two util files in place, one containing util for logger only (to avoid this).
This is done to remove the logger's dependency on util and merge those two util files.

Reply:
Then remove that unused method

Reply:
done